src/site/content/en/blog/maskable-icon/index.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Better PWA icons with maskable icons
```

Comment:
```suggestion
title: Adaptive icon support in PWAs with maskable icons
```
You'll capture more SEO traffic by including `adaptive icons` in your title. Title wording seems to have a big impact on documentation SEO. This phrasing is also more precise.

I don't know this domain so you can ignore this if it's technically inaccurate.